### PR TITLE
Fixed - Directory separator issue on windows for product import file …

### PR DESCRIPTION
**Pull request by ughimire**
…link issue fixed
woo

### CODE REVIEW

**Comment by shivapoudel** [comment]
@claudiulodro Me to having the same issue.

File: includes/admin/importers/class-wc-product-csv-importer-controller.php [hunk 1]
```diff
@@ -128,7 +128,7 @@ public function get_next_step_link( $step = '' ) {
 
 		$params = array(
 			'step'            => $keys[ $step_index + 1 ],
-			'file'            => $this->file,
+			'file'            => str_replace( DIRECTORY_SEPARATOR, "/", $this->file ),
```
Comment (claudiosanches) [inline includes/admin/importers/class-wc-product-csv-importer-controller.php]: Should use single quotes.

**Comment by claudiulodro** [comment]
Makes sense. This should solve the problem. Just have to use single quotes and we're good to merge.

File: includes/admin/importers/class-wc-product-csv-importer-controller.php (hunk 1 above)
Comment (ughimire) [inline includes/admin/importers/class-wc-product-csv-importer-controller.php]: okey :)

**Comment by ughimire** [comment]
Closed in favor of #15550

**Pull request closed without merging**